From 27303b514be8f1b310fced9cc1406948cc6df184 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:17 -0400
Subject: [redhat] kabi: Adding symbol find_get_page (mm/filemap.c)

Message-id: <6985abd5a15f680a54ebf26144e5e9e5cc2218be.1410510773.git.poros@redhat.com>
Patchwork-id: 96030
O-Subject: [PATCH RHEL7.1 077/233] kabi: Adding symbol find_get_page (mm/filemap.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Find and get a page reference.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/find_get_page
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/find_get_page
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/find_get_page

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/find_get_page b/redhat/kabi/kabi-module/kabi_ppc64/find_get_page
new file mode 100644
index 0000000..3bbca1a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/find_get_page
@@ -0,0 +1,2 @@
+#1-
+0xf04d1604 find_get_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/find_get_page b/redhat/kabi/kabi-module/kabi_s390x/find_get_page
new file mode 100644
index 0000000..0762f58
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/find_get_page
@@ -0,0 +1,2 @@
+#1-
+0xd6285b82 find_get_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/find_get_page b/redhat/kabi/kabi-module/kabi_x86_64/find_get_page
new file mode 100644
index 0000000..ac00f62
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/find_get_page
@@ -0,0 +1,2 @@
+#1-
+0xadeacf60 find_get_page vmlinux EXPORT_SYMBOL
-- 
1.7.1